core/domain/user_services_test.py

```diff
@@ -259,40 +258,104 @@ def test_get_user_id_from_username(self):
         self.assertIsNone(
             user_services.get_user_id_from_username('fakeUsername'))
 
-    def test_get_user_settings_by_gae_id(self):
-        gae_id = 'gae_id'
-        user_models.UserSettingsModel(
-            id='user_id',
-            gae_id=gae_id,
-            email='user@example.com',
-            username='username',
-        ).put()
-        user_settings_model = user_models.UserSettingsModel.get_by_gae_id(
-            gae_id)
-        user_settings = user_services.get_user_settings_by_gae_id(gae_id)
-        self.assertEqual(user_settings_model.id, user_settings.user_id)
-        self.assertEqual(user_settings_model.email, user_settings.email)
-        self.assertEqual(user_settings_model.username, user_settings.username)
-        self.assertIsNone(user_services.get_user_settings_by_gae_id('id_x'))
-
-    def test_get_user_settings_by_gae_id_strict(self):
-        gae_id = 'gae_id'
-        user_models.UserSettingsModel(
-            id='user_id',
-            gae_id=gae_id,
-            email='user@example.com',
-            username='username',
-        ).put()
-        user_settings_model = user_models.UserSettingsModel.get_by_gae_id(
-            gae_id)
-        user_settings = user_services.get_user_settings_by_gae_id(
-            gae_id, strict=True)
-        self.assertEqual(user_settings_model.id, user_settings.user_id)
-        self.assertEqual(user_settings_model.email, user_settings.email)
-        self.assertEqual(user_settings_model.username, user_settings.username)
+    def test_user_settings_by_gae_id_with_user_auth_disabled_is_not_none(self):
+        with self.swap(feconf, 'ENABLE_USER_AUTH_MODEL', False):
+            gae_id = 'gae_id'
+            # TODO(#10178): Remove gae_id attribute from UserSettingsModel below
+            # when feconf.ENABLE_USER_AUTH_MODEL flag has been set True.
+            user_models.UserSettingsModel(
+                id='user_id',
+                gae_id=gae_id,
+                email='user@example.com',
+                username='username'
+            ).put()
+            user_settings_model = user_models.UserSettingsModel.get_by_gae_id(
+                gae_id)
+            user_settings = user_services.get_user_settings_by_gae_id(gae_id)
+            self.assertEqual(user_settings_model.id, user_settings.user_id)
+            self.assertEqual(user_settings_model.email, user_settings.email)
+            self.assertEqual(
+                user_settings_model.username, user_settings.username)
+            self.assertIsNone(
+                user_services.get_user_settings_by_gae_id('id_x'))
+
+    def test_user_settings_by_auth_model_with_user_auth_enabled_not_none(self):
```

Comment:
test_user_settings_by_auth_model_with_user_auth_enabled_is_not_none, here and elsewhere